package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "scripts": {
     "precommit": "lint-staged",
-    "prepare": "lerna bootstrap",
+    "prepare": "npm run bootstrap",
```

Comment:
This edit updates npm `prepare` script to reuse the existing `bootstrap` script, in order to support the `npm ci` script.

Reply:
I'm fine with this change, but how does using `npm ci` affect this?

Reply:
Yeah, I've had other issues with the `--hoist` flag. I think we'll need to make sure it works for all use-cases before changing this, though this might be the right way to go. If it is, we should remove the `bootstrap` run script because it would be duplicative with `prepare`